corehq/apps/userreports/reports/specs.py

```diff
@@ -47,6 +48,20 @@
     'year': YearColumn,
 }
 
+ES_AGG_MAP = {
+    'avg': aggregations.AvgAggregation,
+    'count': aggregations.ValueCountAggregation,
+    'count_unique': aggregations.CardinalityAggregation,
+    'min': aggregations.MinAggregation,
+    'max': aggregations.MaxAggregation,
+    'sum': aggregations.SumAggregation,
+    'simple': lambda x, y: None,  # this is not an aggregation
```

Comment:
for an aggregated query `simple` is what we're aggregating over. (`top_agg` in the data source `get_data`) the rest will be subaggregations.